### PR TITLE
Time fix

### PR DESCRIPTION
**Pull request by paigerube14**
### Description
Selecting container if not specified in config for time scenarios

### Fixes
https://github.com/cloud-bulldozer/kraken/issues/177

### CODE REVIEW

File: kraken/time_actions/common_time_functions.py [hunk 1]
```diff
@@ -29,6 +30,20 @@ def node_debug(node_name, command):
     return response
 
 
+def get_container_name(pod_name, namespace, container_name=""):
+    container_names = runcommand.invoke(
+        'oc get pods %s -n %s -o jsonpath="{.spec.containers[*].name}"' % (pod_name, namespace)
```
Comment (chaitanyaenr) [inline kraken/time_actions/common_time_functions.py:35]: Maybe use read_namespaced_pod function from the kubernetes python client library to get the list of containers in pod? Thoughts?

Comment (paigerube14) [reply kraken/time_actions/common_time_functions.py]: Should be updated now, PTAL

File: kraken/time_actions/common_time_functions.py [hunk 2]
```diff
@@ -79,23 +94,45 @@ def skew_time(scenario):
...
+                        "Couldn't reset time on container %s in pod %s in namespace %s"
+                        % (selected_container_name, pod[0], pod[1])
+                    )
+                    sys.exit(1)
+                pod_names[pod_counter].append(selected_container_name)
             else:
-                pod_exec(pod, skew_command, scenario["namespace"])
+                selected_container_name = get_container_name(pod, scenario["namespace"], container_name)
+                pod_exec(pod, skew_command, scenario["namespace"], selected_container_name)
+                if pod_exec_response is False:
```
Comment (chaitanyaenr) [inline kraken/time_actions/common_time_functions.py:112]: Think we missed capturing the pod_exec_response from the pod_exec command.

Comment (paigerube14) [reply kraken/time_actions/common_time_functions.py:112]: Should be fixed now, PTAL

**Comment by chaitanyaenr** [comment]
@paigerube14 should we add the container_name field to the sample scenario config - https://github.com/cloud-bulldozer/kraken/blob/master/scenarios/time_scenarios_example.yml? Thoughts?

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**